app/controllers/result_filters/provider_controller.rb

```diff
@@ -4,7 +4,12 @@ class ProviderController < ApplicationController
 
     def new
       if params[:query].blank?
-        flash[:error] = [I18n.t("location_filter.fields.provider")]
+        flash[:error] = [I18n.t("location_filter.fields.provider"), I18n.t("location_filter.errors.blank_provider")]
```

Comment:
I think it would work without `I18n`,  which would make the lines shorter.

```suggestion
        flash[:error] = [t("location_filter.fields.provider"), t("location_filter.errors.blank_provider")]
```
It's a massive nitpick 🙈 and I don't have a strong preference for either so definitely not a blocker.

Reply:
it should be noted at this point the 2 are not identical so aren't always interchangeable. https://github.com/rails/rails/blob/master/actionview/lib/action_view/helpers/translation_helper.rb#L19

Reply:
But in this case is probably fine to change